src/routes/api/classes.ts

```diff
@@ -1,12 +1,12 @@
 import * as express from 'express';
 const router = express.Router();
-import ClassController from '../../controllers/api/classController';
+import * as ClassController from '../../controllers/api/classController';
```

Comment:
Instead of doing this, would it make more sense to adjust ClassController to export a default object that contains the methods?